tasks.py

```diff
@@ -90,6 +90,12 @@ def adminserver(port=8001):
     cmd = '{} python manage.py runserver {} --nothreading'.format(env, port)
     run(cmd, echo=True, pty=True)
 
+# TODO - refactor? It is functional. e.g. 'invoke admin_task createsuperuser'
+@task
+def admin_task(args):
```

Comment:
That is in my https://github.com/CenterForOpenScience/osf.io/pull/4691 it only works for one argument, otherwise the invoke expects to move on to a new invoke task.